mixer/template/edge/template_handler_service.proto

```diff
@@ -93,44 +93,46 @@ message HandleEdgeRequest {
 
 
 
+
+
 // Contains instance payload for 'edge' template. This is passed to infrastructure backends during request-time
 // through HandleEdgeService.HandleEdge.
 message InstanceMsg {
 
     // Name of the instance as specified in configuration.
     string name = 72295727;
     
-    // Timestamp
+    // Timestamp of the edge
```

Comment:
It was from another PR. I ran go generate across all of mixer, which tends to lead to this problem. I think we should still commit the latest go generated artifacts, instead of keeping them out of sync.